index.html

```diff
@@ -29,5 +29,106 @@ <h4>let's connect</h4>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot2.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot3.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    
+        <section class="skills">
+            <img class="sample" src="./images/Snapshoot4.png" alt="project snapshoot">
+            <header class="project-header">
+                <h2>tonic</h2>
+                <ul class="tags">
+                    <li class="diff-li">canopy</li>
+                    <li>back end dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="set-stack">
+                    <li class="stack">html</li>
+                    <li class="stack">css</li>
+                    <li class="stack">javascipt</li>
+                </ul>
+                <button class="btn" type="submit">see project</button>
+            </header>
+        </section>
+    </div>
```

Comment:
- [ ] Kindly wrap this container of your projects into a `section` tag in order to maintain the good usage of semantics and it will make our work look more professional. :wink: